### PR TITLE
contrib/google.golang.org/grpc: use naming schema

### PR DESCRIPTION
**Pull request by rarguelloF**
### What does this PR do?

Implement naming schema for `contrib/google.golang.org/grpc` and `contrib/google.golang.org/grpc.v12` integrations.

Additional changes:
- Fix a bug where service name was not being set for `UnaryClientInterceptor` in `contrib/google.golang.org/grpc.v12`.
- Add missing `span.kind` tags for `NewClientStatsHandler` and `NewServerStatsHandler`.

### Motivation

Improve service and span names.

### Reviewer's Checklist

- [x] Changed code has unit tests for its functionality.
- [x] If this interacts with the agent in a new way, a system test has been added.

### CODE REVIEW

**Review by ajgajg1134 — APPROVED** [review]
Looks good!

**Comment by dianashevchenko** [comment]
@rarguelloF have you refactored the test part as we discussed previously?

**Comment by rarguelloF** [comment]
@dianashevchenko yes! it's the last commit, sorry I forgot to ping you

File: contrib/google.golang.org/grpc/grpc_test.go [hunk 1]
```diff
@@ -1029,6 +1022,101 @@ func TestCustomTag(t *testing.T) {
...
+	t.Run("ServiceName", namingschematest.NewServiceNameTest(genSpans, "", wantServiceNameV0))
+	t.Run("SpanName", namingschematest.NewOpNameTest(genSpans, assertOpV0, assertOpV1))
+}
+
+func TestClientNamingSchema(t *testing.T) {
+	genSpans := getGenSpansFn(true, false)
+	assertOpV0 := func(t *testing.T, spans []mocktracer.Span) {
+		require.Len(t, spans, 4)
+		for i := 0; i <= 3; i++ {
+			assert.Equal(t, "grpc.client", spans[i].OperationName())
```
[the remark this answers is not in the document]
Comment (rarguelloF) [reply contrib/google.golang.org/grpc/grpc_test.go]: changed!

**Review by dianashevchenko — APPROVED** [review]

**Pull request merged**